vendor/github.com/opencontainers/runtime-spec/specs-go/config.go

```diff
@@ -6,26 +6,24 @@ import "os"
 type Spec struct {
 	// Version of the Open Container Runtime Specification with which the bundle complies.
 	Version string `json:"ociVersion"`
-	// Platform specifies the configuration's target platform.
-	Platform Platform `json:"platform"`
 	// Process configures the container process.
-	Process Process `json:"process"`
+	Process *Process `json:"process,omitempty"`
```

Comment:
I think this change will mean you'll need an `initProcess` function and some `nil` checks (e.g. [here][1]).  You'll also need similar changes for `Root` → `*Root`.

[1]: https://github.com/opencontainers/runtime-tools/blob/0d35fc8437ec8292cd7065e332000c584f7024c5/validate/validate.go#L214-L215